### ISSUE TITLE
asan only shows address, no source line info

### ISSUE THREAD

**Issue by pangqingyuan1991** [google/sanitizers#1038]
My project has many thridparty libraries and our internal source code. I am sure I have compiled internal source code with -fsanitize=address -fno-omit-frame-pointer -g -O0
But when asan report some errors, it only prints executable+address, no source line.
I can use addr2line to get the source line with address. But it is not convenient. 

My another project works well. And it doesn't need ASAN_SYMBOLIZER_PATH. Both projects use the same gcc version 5.4.0

What may I missing?

**Comment by vitalybuka** [comment]
do you have llvm-symbolizer on the PATH ?

**Comment by pangqingyuan1991** [comment]
> do you have llvm-symbolizer on the PATH ?

No. But since my another program works well. I don't think llvm-symbolizer is a must.

**Comment by vitalybuka** [comment]
It can use addr2line if you set env variable
ASAN_OPTIONS=allow_addr2line=1 
See: https://github.com/google/sanitizers/wiki/SanitizerCommonFlags 

Also you can check if the another program has __asan_default_options(), e.g.:
extern "C" const char* __asan_default_options() {
  return "allow_addr2line=true";
}

**Comment by vitalybuka** [comment]
Actually with gcc it suppose to use libbacktrace https://github.com/llvm-project/llvm-project-20170507/blob/f39a516d4db4f1be01608683ebe51d400c0989cf/compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_libbacktrace.cc#L150

**Comment by pangqingyuan1991** [comment]
> Actually with gcc it suppose to use libbacktrace https://github.com/llvm-project/llvm-project-20170507/blob/f39a516d4db4f1be01608683ebe51d400c0989cf/compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_libbacktrace.cc#L150

Yes. I use gcc 5.3.1
